manifests/configfile.pp

```diff
@@ -52,7 +56,9 @@
     default  => 'file',
   }
 
-  file { "${config_dir}/${title}.conf":
+  $filename = "${title}${file_extension}"
```

Comment:
should this be `"${title}.${file_extension}"`, or should we maybe enforce that `$file_extension` starts with a dot?

Reply:
This is to allow to include any file irregardless of the extension

Reply:
Though, I see the point.. maybe it should be `Variant[Enum[''], Pattern[/^\..*$/]]` to allow both empty string and anything starting with a dot. I hope nobody would want to use underscore or dash here 🤞🏻.

Reply:
Then, maybe it's better to rename it from "extension" to "suffix", as it reflects the behavior better.